src/libcore/cell.rs

```diff
@@ -187,6 +187,29 @@ use ops::{Deref, DerefMut, CoerceUnsized};
 use ptr;
 
 /// A mutable memory location.
+/// # Examples
```

Comment:
like here? https://github.com/rust-lang/rust/pull/43423/commits/82860753463314e6a1b94f1f97d4d9c4effc0742

Because i put it at first place and it was the cause of CI failure (something related to tidy check)